modules/lvgl/lvgl.c

```diff
@@ -253,4 +275,6 @@ static int lvgl_init(void)
 	return 0;
 }
 
+#ifdef CONFIG_LV_Z_AUTO_INIT
```

Comment:
I'm wondering if would be good also to make the priority of the auto init something that can be configured.

Reply:
Sure why not does not hurt to have this in case you have some intricate setup where you need to manage init priorities within the application level. Added this.